packages/manager/modules/pci/src/projects/project/inactive/inactive.html

```diff
@@ -0,0 +1,44 @@
+<div class="oui-modal oui-modal_shadow">
+    <div class="oui-modal__header">
+        <button
+            class="oui-icon oui-icon-close_thin oui-modal__close-button"
+            type="button"
+            aria-label="Exit"
+            data-ng-click="$ctrl.goBack()"
+        ></button>
+    </div>
+    <div class="oui-modal__body">
+        <span
+            class="oui-icon oui-icon-warning_circle oui-icon_bicolor"
+            aria-hidden="true"
+        ></span>
+        <div class="oui-modal__body-text">
+            <h2 class="oui-modal__title" data-ng-bind="::$ctrl.title"></h2>
+            <p data-ng-bind="::$ctrl.description"></p>
+        </div>
+    </div>
+    <div class="oui-modal__footer">
+        <oui-button
+            data-ng-if="$ctrl.projectData.isPendingDebt && !$ctrl.projectData.isSuspended"
+            data-variant="secondary"
+            data-on-click="$ctrl.goToProject({ project_id: $ctrl.projectId })"
+        >
+            <span data-translate="pci_projects_project_view"></span>
+        </oui-button>
+        <oui-button
+            class="ml-3"
+            data-ng-if="$ctrl.projectData.isPendingDebt"
```

Comment:
I would expect `isPendingDebt` to be called on a debt not a `project`
```suggestion
            data-ng-if="$ctrl.projectData.hasPendingDebt"
```

Reply:
Changed :)